build/gulp/tasks/test-projects.tsx

```diff
@@ -61,9 +96,6 @@ task('test:projects:cra-ts', async () => {
   const logger = log('test:projects:cra-ts')
   const scaffoldPath = paths.base.bind(null, 'build/gulp/tasks/test-projects/cra')
 
-  const packageFilename = await publishPackage()
-  logger(`✔️Package was published: ${packageFilename}`)
```

Comment:
it was helpful for debugging purposes to have paths (previously only the path for 'react' package) being displayed on the console - could we preserve this (in the corresponding step)?